website/client/src/assets/scss/button.scss

```diff
@@ -1,15 +1,20 @@
 .btn {
+  height: 2rem;
```

Comment:
is it necessary to specify these absolute heights (also for :content)? I think it might be the cause of some of the buttons weirdness currently on delta and I'm not sure it's necessary: once paddings, borders and line height for the text is defined it should give css all the necessary info about the size of the element without having to specify the width or height

Reply:
I'll check the changes with heights, if those aren't needed I'll remove them again